instrumentation/apache-httpclient/apache-httpclient-4.3/library/README.md

```diff
@@ -0,0 +1,50 @@
+# Library Instrumentation for Apache HttpClient version 4.3 and higher
+
+Provides OpenTelemetry instrumentation for [Apache HttpClient](https://hc.apache.org/httpcomponents-client-ga/).
+
+This instrumentation generates CLIENT spans for each HTTP request.
```

Comment:
Perhaps `HTTP client spans and metrics`. I'd drop the `for each HTTP request` part.

okhttp uses
> Provides OpenTelemetry instrumentation for [okhttp3](https://square.github.io/okhttp/).